### ISSUE TITLE
Unresolved reference warnings in Pipeline

### ISSUE THREAD

**Issue by fenil-eic** [tSQLt-org/tSQLt#149]
**tSQLt Version:** 1.0.5873.27393
**SQL:** Azure SQL

**Description:**
We've integrated tSQLt in azure devops pipeline. It is working fine and executing the test cases, but shows some warnings regarding reference. There were total 19 warnings, from that we're unable to fix 2 warnings after trying several solutions.
We're getting same warnings in database projects as well which is displaying in Visual Studio.

**Warnings:**
1. Warning SQL71502: Procedure: [tSQLt].[InstallExternalAccessKey] has an unresolved reference to object [tSQLtExternalAccessKey].
2. Warning SQL71502: Procedure: [tSQLt].[RemoveExternalAccessKey] has an unresolved reference to object [tSQLtExternalAccessKey].

We're looking for the solution to fix this warnings, can anyone has solution to fix those 2 warnings?

![tsqlt-warnings](https://user-images.githubusercontent.com/97660535/149724641-365f11f5-6419-49fd-a1c3-5dff1723b752.png)
![tsqlt-warnings-2](https://user-images.githubusercontent.com/97660535/149724656-7d98de9c-4b27-4545-a7af-5656a59ee43f.png)

Tags: @tSQLt @tSQLt-robot

**Comment by mbt1** [comment]
@fenil-eic, thank you for submitting this. We already spent a couple of hours looking into this but ran into a few roadblocks, due to the age of that code. 
Before we invest more time, can you let us know if this is a blocking issue for your pipeline?

**Comment by fenil-eic** [comment]
Yes. As per the requirements, we have to clear all the warnings from the pipeline and we're unable to fix it so its blocker for us.
One more issue we're facing due to this reference error is, we're unable to take backup of the azure database. We're facing below error:

> One or more unsupported elements were found in the schema used as part of a data package.
> Error SQL71501: Error validating element [tSQLt].[Private_ScriptIndex]: Function: [tSQLt].[Private_ScriptIndex] contains an unresolved reference to an object. Either the object does not exist or the reference is ambiguous because it could refer to any of the following objects: [tSQLt].[Private_SysIndexes].[I] or [tSQLt].[Private_SysIndexes].[is_hypothetical].
> Error SQL71501: Error validating element [tSQLt].[Private_

![db-backup-error](https://user-images.githubusercontent.com/97660535/150070621-c175a587-1650-4019-b1db-029845f5e97c.png)

**Comment by fenil-eic** [comment]
@mbt1 Any updates on it? We're waiting for the solution so we can move further!

**Comment by mbt1** [comment]
@fenil-eic, would you be open to a beta test?

**Comment by fenil-eic** [comment]
@mbt1, Yes we're open for that. Please let us know further steps for same.

**Comment by lizbaron** [comment]
Hi @fenil-eic,

We'd like to send you the beta version via email. Could you please contact us here and reference the ticket number (149)? https://sqlity.net/en/contact/ 

Thanks,
Liz

**Comment by fenil-eic** [comment]
@lizbaron email sent on given id.

**Comment by fenil-eic** [comment]
Hello @lizbaron, @mbt1 

Thank you for helping out, warnings has been fixed. But still we're facing the issue while taking backup/export the database (2nd issue which I've mentioned in comment).

Currently we're facing below error because of tSQLt, can you please help us in fixing it?

![image](https://user-images.githubusercontent.com/97660535/153042936-a443eb3b-6f9c-41aa-aa4d-ad9503e44782.png)
